packages/graphql-codegen-cli/src/utils/debugging.ts

```diff
@@ -6,21 +6,17 @@ let queue: Array<{
 }> = [];
 
 export function debugLog(message: string, ...meta: any[]) {
-  if (!process.env.GQL_CODEGEN_NODEBUG && process.env.DEBUG !== undefined) {
```

Comment:
Will it still work if we do DEBUG=1 as env variable?

Reply:
@saihaj, I'm in favor of dropping support for `DEBUG` and favor the cli flags

do you think it should be a minor then?

Reply:
I think let’s make it work with env variable and drop in next major

Reply:
Okay, then let me refactor the PR